packages/runtime/container-runtime/src/garbageCollectionConstants.ts

```diff
@@ -10,6 +10,9 @@ export const stableGCVersion: GCVersion = 1;
 /** The current version of garbage collection. */
 export const currentGCVersion: GCVersion = 2;
 
+/** This undocumented GC Option (on ContainerRuntime Options) allows an app to disable enforcing GC on old documents by incrementing this value */
+export const gcTombstoneEnforcementValueOptionName = "gcTombstoneEnforcementValue";
```

Comment:
gcTombstoneGeneration?

Reply:
Oooh yeah gotta change this one.  Thanks.